extensions.json

```
@@ -595,6 +599,10 @@
       "download": "https://github.com/kasecato/vscode-docomment/releases/download/v0.1.18/docomment-0.1.18.vsix",
       "version": "0.1.18"
     },
+    {
+      "id": "kaiwood.endwise",
+      "repository" : "https://github.com/kaiwood/vscode-endwise"
```

Since this repository offers dedicated release tags, I think it would be best to use them:

```suggestion
      "repository" : "https://github.com/kaiwood/vscode-endwise",
      "version": "1.4.2",
      "checkout": "v1.4.2"
```